src/components/keypad/pre-algebra-page.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Keys from "../../data/key-configs";
+import {
+    KeypadPageContainer,
+    KeypadButton,
+    SecondaryKeypadButton,
+    KeypadActionButton,
+    PlaceHolderButton,
+} from "./keypad-page-items";
+
+import type {KeyConfig} from "../../data/key-configs";
+
+type Props = {|
+    onClickKey: (keyConfig: string) => void,
+|};
+type State = {};
+export default class NumericInputPage extends React.Component<Props, State> {
+    render() {
+        const {onClickKey} = this.props;
+        return (
+            <KeypadPageContainer>
+                {/* Row 1 */}
+                <SecondaryKeypadButton
+                    keyConfig={Keys.EXP_2}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.EXP}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.SQRT}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.RADICAL}
+                    onClickKey={onClickKey}
+                />
+
+                <SecondaryKeypadButton
+                    keyConfig={Keys.LEFT_PAREN}
+                    onClickKey={onClickKey}
+                />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.RIGHT_PAREN}
+                    onClickKey={onClickKey}
+                />
+                {/* Row 2 */}
+                <PlaceHolderButton />
+                <PlaceHolderButton />
+                <PlaceHolderButton />
+                <SecondaryKeypadButton
+                    keyConfig={Keys.X}
+                    onClickKey={onClickKey}
+                    style={{
+                        gridColumn: 5,
+                    }}
```

Comment:
Why do we need to specify the `gridColumn` **and** have `PlaceholderButton`s?

Reply:
Placeholder button is a space where there will sometimes be a button. These "skips" are where there will never be a button